modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -96,6 +96,32 @@ as well as individual components.
 
 To recover a node, follow the instructions provided in xref:manage-nodes.adoc[Add or Remove a Node from a Runtime Fabric].
 
+== Troubleshoot DNS Resolution Issues
```

Comment:
I'm unsure about this change. Some things I find concerning:
 - the kubectl command we're exposing to the user. we should refrain from publicly documenting kubectl commands.
 - the example of a tokenization service.
 - the log output itself. it's very hard to discern what to look for.

If we need to provide DNS resolution troubleshooting, we should be able to do this better, perhaps via an `rtfctl` command.
cc @sarjeet2013

Reply:
Agree. We should take out this portion I pointed below.

Reply:
Removed this.